src/main/java/cn/nukkit/level/EnumLevel.java

```diff
@@ -18,9 +19,38 @@ public Level getLevel() {
 
     public static void initLevels() {
         OVERWORLD.level = Server.getInstance().getDefaultLevel();
+
+        // attempt to load the nether world
+        if (Server.getInstance().isNetherAllowed() && !Server.getInstance().loadLevel("nether")) {
+
+            // Nether is allowed, and not found, create the default nether world
+            Server.getInstance().getLogger().info("No level called \"nether\" found, creating default nether level.");
+
+            // Generate seed for nether and get nether generator
+            long seed = System.currentTimeMillis();
+            Class<? extends Generator> generator = Generator.getGenerator("nether");
+
+            // Generate the nether world
+            Server.getInstance().generateLevel("nether", seed, generator);
+
+            // Finally, load the level if not already loaded and set the level
+            if (!Server.getInstance().isLevelLoaded("nether")) {
+                Server.getInstance().loadLevel("nether");
+            }
+
+        }
+
         NETHER.level = Server.getInstance().getLevelByName("nether");
-        if (NETHER.level == null)   {
-            Server.getInstance().getLogger().alert("No level called \"nether\" found! Nether functionality will be disabled.");
+
+        if (NETHER.level == null && Server.getInstance().isNetherAllowed())   {
+            // Nether is allowed, and not found, alert the user
+            Server.getInstance().getLogger().alert("No level called \"nether\" found or nether is disabled in server properties! Nether functionality will be disabled.");
+        } else if (NETHER.level == null && !Server.getInstance().isNetherAllowed()) {
+            // don't load nether world
+            Server.getInstance().getLogger().alert("No level called \"nether\" found or nether is disabled in server properties! Nether functionality will be disabled.");
+        } else if (NETHER.level != null && !Server.getInstance().isNetherAllowed()) {
+            // don't load the nether world
```

Comment:
this whole block is pretty redundant, merge all of those checks into one `if`

Reply:
Fixed the if statement as we only load the nether if it is allowed in server properties, I am only checking to see if it is null.